include/_mgp.hpp

```diff
@@ -290,6 +290,8 @@ mgp_list *list_make_empty(size_t capacity, mgp_memory *memory) {
 
 mgp_list *list_copy(mgp_list *list, mgp_memory *memory) { return MgInvoke<mgp_list *>(mgp_list_copy, list, memory); }
 
+mgp_value *list_value_copy(mgp_list *val, mgp_memory *memory) { return mgp::value_make_list(list_copy(val, memory)); }
```

Comment:
The `type_value_copy` functions are actually not required... 🤔

Reply:
Do you mean that the `type_value_copy` funcions should be removed in the next iteration of the PR?

Reply:
Do you mean that the `type_value_copy` funcions should be removed in the next iteration of the PR?